docs/standard-library/built-in-decorators.md

```diff
@@ -305,6 +305,25 @@ Invalid,
 ```
 
 
+### `@list` {#@list}
```

Comment:
that is unfortunate that we get the doc now that it is deprecated

Reply:
yeah I guess that is dependent on the design meeting that was inconclusive last week :(